apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -253,6 +153,114 @@ struct GroupMemberInfoView: View {
         }
         .modifier(ThemedBackground(grouped: true))
     }
+    
+    @ViewBuilder
+    private func sections(_ member: GroupMember) -> some View {
```

Comment:
moved without changes

Reply:
why does it need to move?

Reply:
without moving, it weirdly works differently...